integrationTests/testProcessorNode.go

```diff
@@ -1120,7 +1124,7 @@ func (tpn *TestProcessorNode) syncMetaNode(nonce uint64) error {
 		header,
 		body,
 		func() time.Duration {
-			return time.Second * 2000
+			return time.Second * 2000000
```

Comment:
This will probably slow down testing a lot.

Reply:
done